### DISCUSSION TITLE
request help: disable prometheus for specific routes when enable it as a global rule

### DISCUSSION THREAD

**Discussion by suninuni** [apache/apisix#5051] [Q&A]
### Issue description

I want enable prometheus plugin as a global rule for metics, but I don't want to see some paths such as healthz。

### Environment

- apisix version (cmd: `apisix version`): 2.7

**Comment by tzssangglass** [comment]
> paths such as healthz

you mean health check? show some about this.

**Reply by suninuni** [reply]
For example, health check of Alibaba Cloud SLB.

<img width="918" alt="1631523319(1)" src="https://user-images.githubusercontent.com/7345529/133055162-5b4ea445-bc19-4d92-bd85-8216ae3f33f9.png">

**Reply by tzssangglass** [reply]
This is a common problem and there is no good way to solve it. It is common practice to filter the prometheus exporter information after it has been collected.
